CONTRIBUTING.md

```diff
@@ -45,10 +47,31 @@ cargo clippy --workspace
 [`clippy`]: https://github.com/rust-lang/rust-clippy
 
 ### Tests
+
 The test suite include unit test and integration tests.
 
+#### Test feature `xmllint`
+
+`transit_model` is capable of exporting NeTEx France format.
+The tests are automatically verifying that the produced files are matching the
+NeTEx specification.
+
+For that, tests are using the tool `xmllint` which can be installed on Debian
+with the package `libxml2-utils`.
+Therefore, these tests are run only if this feature is activated.<br>
+Tests also depend on NeTEx specification that are imported as a git submodule.
+
+To install xmllint and submodules:
+```sh
+git submodule update --init --recursive
+apt install libxml2-utils
+```
```

Comment:
```suggestion
#### Test feature `xmllint`

`transit_model` is capable of exporting NeTEx France format.<br>
Integration tests verify that the conversion produces files in accordance with the NeTEx specification.

For that, tests are using the tool `xmllint` which can be installed on Debian
with the package `libxml2-utils`.
Therefore, these tests are run only if this feature is activated.<br>
Tests also depend on NeTEx specification that are imported as a git submodule.

To install xmllint and submodules:
```sh
git submodule update --init --recursive
apt install libxml2-utils
```
```
If you want to validate your product (NeTEx France) file you can use the following command:
```sh
xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
```